### PR TITLE
add yup token to `prices_ethereum_tokens.sql`

### PR DESCRIPTION
**Pull request by andrei0x309**
# Thank you for contributing to Spellbook!

Thank you for taking the time to submit code in Spellbook. A few things to consider:

- If you are a first-time contributor, please sign the CLA by copy & pasting **_exactly_** what the bot mentions in PR comment
- Refer to [docs](#spellbook-contribution-docs) section below to answer questions
- Dune team will review submitted PRs as soon as possible

## Spellbook sub-projects

To prepare for Spellbook to scale in size, Dune team has started to pilot sub-projects:

- For a quick overview, see [the GH discussion on the topic](https://github.com/duneanalytics/spellbook/discussions/5238)
  - Please ask questions and continue the conversation there
- The first spell lineage to move: `tokens.erc20` and all upstream spells which help build it
  - **_Note_**: Please navigate to [tokens](/tokens) root directory to find tokens objects

## Spellbook contribution docs

The [docs](docs) directory has been implemented to answer as many questions as possible. Please take the time to reference each `.md` file within this directory to understand how to efficiently contribute & why the repo is designed as it is 🪄

Example questions to be answered:

- What does each property in the [model config block](/docs/models/model_config_block.md) mean?
- What is the [CI test](/docs/ci_test/ci_test_overview.md) attached to PRs and how can I best utilize it?
- What are the Spellbook [best practices](/docs/general/best_practices.md)?

Please navigate through the [docs](/docs) directory to find as much info as you can.

**Note**: happy to take PRs to improve the docs, let us know 🤝

### CODE REVIEW

**Comment by andrei0x309** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by andrei0x309** [comment]
recheck

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1651,7 +1651,8 @@ FROM
     ('oeth-origin-ether','ethereum','OETH',0x856c4Efb76C1D1AE02e20CEB03A2A6a08b0b8dC3, 18),
     ('ethx-stader-ethx','ethereum','ETHx',0xA35b1B31Ce002FBF2058D22F30f95D405200A15b, 18),
     ('rseth-rseth','ethereum','rsETH',0xa1290d69c65a6fe4df752f95823fae25cb99e5a7, 18),
-    ('pufeth-pufeth','ethereum','pufETH',0xD9A442856C234a39a81a089C06451EBAa4306a72, 18)
+    ('pufeth-pufeth','ethereum','pufETH',0xD9A442856C234a39a81a089C06451EBAa4306a72, 18),
+    ('yup-yup','ethereum','YUP',0x69bBC3F8787d573F1BBDd0a5f40C7bA0Aee9BCC9, 18)
```
Comment (Hosuke) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1655]: API: https://api.coinpaprika.com/v1/coins/yup-yup
SCAN: https://etherscan.io/token/0x69bBC3F8787d573F1BBDd0a5f40C7bA0Aee9BCC9

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @andrei0x309

**Pull request merged**